web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -2019,10 +2024,13 @@ public VoidResponse adminClose(@Required("id") @PathParam("id") String id) {
                     RunningQuery query = adminGetQueryById(id);
                     close(query);
                 } catch (Exception e) {
-                    // if this is a query that is in the old query cache, then we have already successfully closed this query so ignore
-                    if (oldQueryCache.getIfPresent(id) == null) {
+                    log.warn("Failed to adminClose " + id + ", checking if closed previously");
```

Comment:
adminClose is for administrators closing a user's query. I would think if the administrator submits that request, and the user query happened to complete/return a 204 immediately before, we would still want this request to fail so the administrator can look into what happened.

Reply:
agreed...will do